torch/_tensor_docs.py

```diff
@@ -2848,15 +2848,26 @@ def callable(a, b) -> number
 
 add_docstr_all('size',
                r"""
-size() -> torch.Size
+size(dim) -> torch.Size or int
 
-Returns the size of the :attr:`self` tensor. The returned value is a subclass of
-:class:`tuple`.
+Returns the size of the :attr:`self` tensor.
+
+:class:`torch.Size` of lengths of each dimension is returned when no argument is passed in. 
+The returned value is a subclass of :class:`tuple`. Otherwise, integer value is returned as
```

Comment:
nit: Otherwise, an integer value representing the length of the specified dimension is returned.